script/Arcueid/Day 8/08_00_ARC08_1_1.txt

```diff
@@ -31,27 +33,28 @@ Uh-oh. Not good.
 [sha:639b6408cac49509e372025f5bf3d327de64ca8e]{
 -- Page 36, Offset 19765.
 -- 秋葉の視線はよけいきつくなったようだ。
-It looks like Akiha's gaze is getting even more intense.
+If anything, it seems Akiha's gaze got even sharper.
 }
 [sha:a7f6007faac8f2a39a28d654cfa224196fce5458]{
 -- Page 36, Offset 19766.
 -- ……辛いな。
-...How harsh.
+...what a bind.
 }
 [sha:ba74f9ac8a30ee5b90b377373e3d33a1f30f431d]{
 -- Page 36, Offset 19767.
 -- これ以上ここで秋葉の無言の圧力を受けていると、間違いなく胃に穴があく。
-If I take in any more of Akiha's silent pressure, I'm definitely going to end up with a hole in my stomach.
+If I'm on the receiving end of much more of this silent pressure,
+ I'm definitely going to end up with a hole in my stomach.
 }
 [sha:064212691815fa43749855eb5c872128b4e00ba8]{
 -- Page 37, Offset 19768.
 -- 「では、おじいさんは朝食を食べに食堂へ」
-"Well then, this old man is going to get some breakfast..."
+"Well then, this old man is going to go get some breakfast..."
```

Comment:
this is something I remember taking note of in the Higurashi MG translation (which translated it the exact same way). I don't think this is how you'd say this in English at all. Maybe something like
"Well then, grandpa needs some breakfast..."
"Well then, a geezer like myself needs some breakfast..."
"In my old age, I'm off to get some breakfast"
etc.